video/out/drm_common.c

```diff
@@ -47,12 +47,18 @@ const struct m_sub_options drm_conf = {
         OPT_STRING_VALIDATE("drm-connector", drm_connector_spec,
                             0, drm_validate_connector_opt),
         OPT_INT("drm-mode", drm_mode_id, 0),
-        OPT_INT("drm-overlay", drm_overlay_id, 0),
+        OPT_INT("drm-osd-plane-id", drm_osd_plane_id, 0),
```

Comment:
Should we not document these options as well? If we do we probably also need to clarify when what plane is use. That only when using drmprime hwdec is the video actually rendered to the video plane &c.

I guess we could consider it internal/primarily for API use due to their confusing nature...